doxy/samples.mapping.txt

```diff
@@ -1,73 +1,72 @@
-# Mapping C++ sample files to Java sample files
-a110x-intr.cxx	A110X_intrSample.java
-a110x.cxx	A110XSample.java
-adc121c021.cxx	ADC121C021Sample.java
-adxl345.cxx	Adxl345Sample.java
-biss0001.cxx	BISS0001Sample.java
-bmpx8x.cxx	BMPX8XSample.java
-buzzer-sound.cxx	Buzzer_soundSample.java
-cjq4435.cxx	CJQ4435Sample.java
-ds1307.cxx	DS1307Sample.java
-enc03r.cxx	ENC03RSample.java
-es08a.cxx	ES08ASample.java
-button.cxx	ButtonSample.java
-groveehr.cxx	GroveEHRSample.java
-groveled.cxx	GroveLEDSample.java
-grovelinefinder.cxx	GroveLineFinderSample.java
-light.cxx	LightSample.java
-grovemoisture.cxx	GroveMoistureSample.java
-relay.cxx	RelaySample.java
-rotary.cxx	RotarySample.java
-grovescam.cxx	GROVESCAMSample.java
-slide.cxx	SlideSample.java
-speaker.cxx	SpeakerSample.java
-vdiv.cxx	VDivSample.java
-water.cxx	WaterSample.java
-grovewfs.cxx	GroveWFSSample.java
-guvas12d.cxx	GUVAS12DSample.java
-h3lis331dl.cxx	H3LIS331DLSample.java
-hcsr04.cxx	HCSR04Sample.java
-hm11.cxx	HM11Sample.java
-hmc5883l.cxx	Hmc5883lSample.java
-htu21d.cxx	HTU21DSample.java
-itg3200.cxx	Itg3200Sample.java
-jhd1313m1-lcd.cxx	Jhd1313m1_lcdSample.java
-joystick12.cxx	Joystick12Sample.java
-lcm1602-i2c.cxx	Lcm1602_i2cSample.java
-ldt0028.cxx	LDT0028Sample.java
-lol.cxx	LoLSample.java
-lsm303.cxx	LSM303Sample.java
-m24lr64e.cxx	M24LR64ESample.java
-max44000.cxx	MAX44000Sample.java
-mic.cxx	MicrophoneSample.java
-mma7455.cxx	MMA7455Sample.java
-mma7660.cxx	MMA7660Sample.java
-mpl3115a2.cxx	MPL3115A2Sample.java
-mpr121.cxx	MPR121Sample.java
-mpu9150.cxx	MPU9150Sample.java
-mq2.cxx	MQ2Sample.java
-mq303a.cxx	MQ303ASample.java
-mq5.cxx	MQ5Sample.java
-groveledbar	GroveLEDBar
-nrf24l01-receiver.cxx	NRF24L01_receiverSample.java
-nrf24l01-transmitter.cxx	NRF24L01_transmitterSample.java
-nunchuck.cxx	NUNCHUCKSample.java
-otp538u.cxx	OTP538USample.java
-ppd42ns.cxx	PPD42NSSample.java
-pulsensor.cxx	PulsensorSample.java
-rfr359f.cxx	RFR359FSample.java
-rotaryencoder.cxx	RotaryEncoderSample.java
-rpr220-intr.cxx	RPR220_intrSample.java
-rpr220.cxx	RPR220Sample.java
-ssd1308-oled.cxx	SSD1308_oledSample.java
-ssd1327-oled.cxx	SSD1327_oledSample.java
-st7735.cxx	ST7735Sample.java
-stepmotor.cxx	StepMotorSample.java
-tm1637.cxx	TM1637Sample.java
-tp401.cxx	TP401Sample.java
-tsl2561.cxx	TSL2561Sample.java
-ttp223.cxx	TTP223Sample.java
-ublox6.cxx	Ublox6Sample.java
-uln200xa.cxx	ULN200XASample.java
-wt5001.cxx	WT5001Sample.java
-yg1006.cxx	YG1006Sample.java
+# Mapping examples across provided languages
```

Comment:
Interesting, I was not aware of this file.  No doubt it is incomplete.  What about C examples?  How are cases handled where a driver has more than one example (urm37 vs. urm37-uart, for example).